credsweeper/app.py

```diff
@@ -78,15 +77,21 @@ def __init__(self,
         self.ml_threshold = ml_threshold
         self.ml_validator = None
 
+    # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
```

Comment:
Why those lines are added? It just for readability..? :thinking:

Reply:
Yes, it helps review large files + some tools like 'meld' use the delimiters like anchors and recognize diff more correctly.

Reply:
Okay.. I agree.